io-backend/src/main/java/agh/io/iobackend/model/User.java

```diff
@@ -0,0 +1,39 @@
+package agh.io.iobackend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "users")
+@Builder
+public class User {
+
+    @Id
+    @SequenceGenerator(
+            name = "user_sequence",
+            sequenceName = "user_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "user_sequence"
+    )
+    @Column(name = "user_id")
```

Comment:
unique? czy to juz jakos z bazie bedzie ogarniete

Reply:
@Id wskazuje że to będzie primary key w tabeli, więc to już bazka powinna sobie ogarnać